cpu/cc3200/periph/gpio.c

```diff
@@ -0,0 +1,393 @@
+/*
+ * Copyright (C) 2019 Ludwig Maximilian Universität
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup        cpu_cc3200
+ * @{
+ *
+ * @file
+ * @brief           GPIO driver implementation
+ *
+ * @author          Wladislaw Meixner <wladislaw.meixner@campus.lmu.de>
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "cpu.h"
+#include "periph/gpio.h"
+#include "vendor/hw_gpio.h"
+#include "vendor/hw_memmap.h"
+#include "vendor/hw_ocp_shared.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#ifdef MODULE_PERIPH_GPIO_IRQ
+/**
+ * @brief   static callback memory
+ */
+static gpio_isr_ctx_t isr_ctx[4][8];
```

Comment:
That should be fine now. While looking at this I thought I is probably wiser in the long run to include the GPIO_BASEA4 since I already reference it in `ports[]`. So I have done just that.  Thanks for the in depth review of the code.